packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -145,8 +146,11 @@ describeFullCompat(`Dehydrate Rehydrate Container Test`, (getTestObjectProvider)
         return getSnapshotTreeFromSerializedContainer(JSON.parse(container.serialize()));
     };
 
-    beforeEach(async () => {
+    beforeEach(async function() {
         provider = getTestObjectProvider();
+        if (compare(provider.driver.version, "0.46.0") === -1 && provider.driver.type !== "odsp") {
```

Comment:
>provider.driver.type !== "odsp") [](http://example.com/codeflow?start=65&length=32)

These test should work for local drvier, right?
May be you should list == "tinylicious" and == "routerlicious" explicitly instead?